src/WordPress/Blueprints/Model/DataClass/ActivatePluginStep.php

```diff
@@ -19,7 +19,7 @@ class ActivatePluginStep implements StepDefinitionInterface
 	 * Plugin slug, like 'gutenberg' or 'hello-dolly'.
 	 * @var string
 	 */
-	public $slug;
+	public $slug = null;
```

Comment:
`null` is the default value for all uninitialized field and is okay to skip from the declaration.

Reply:
This comes from fixing the bug in the model generator. Where it either can
set an inconsequential null in some places or not set defauls "" for
string, false for bool, or 0 for int.

Reply:
I'm not sure what the issue was but I removed these nulls without affecting `""`, `false`, and `0` values in 8ddb3c8b783edc4a03f686b735d0cf8b3dca6f57.